lib/Operation.php

```diff
@@ -73,7 +75,18 @@ protected function buildCommand(string $template, Node $node, string $event, arr
 
 		if (strpos($command, '%n')) {
 			// Nextcloud relative-path
-			$command = str_replace('%n', escapeshellarg($node->getPath()), $command);
+                        $nodeID = -1;
+                        try {
+                                $nodeID = $node->getId();
+                        } catch (InvalidPathException $e) {
+                        } catch (NotFoundException $e) {
+                        }
+                        $config = new Config('config/');
```

Comment:
this should come from \OCP\IConfig (method `getSystemValue`) and ought to be injected in the constructor.

Reply:
Should be resolved. Can you check it?